.changeset/spotty-cobras-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/search-input': patch
+---
+
+Add internal props for temporary reuse of dropdown behavior in other components
```

Comment:
what do you mean by temp?

Reply:
We're planning to create a dropdown component to abstract this behavior in the long run, so we're going to be replacing this pattern pretty soon from what I understand @bruugey

Reply:
Would omit. Let's just say: An internal prop for reuse of dropdown/search behavior in other components.